src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsTableViewCell.swift

```diff
@@ -227,6 +233,45 @@ class HomeStatisticsTableViewCell: UITableViewCell {
 			)
 		}
 	}
+	
+	private func configurePandemicRadarCard(
+		onInfoButtonTap: @escaping CompletionVoid,
+		onAccessibilityFocus: @escaping CompletionVoid,
+		onUpdate: @escaping CompletionVoid
+	) {
+		guard let pandemicLinkCard = cellModel?.linkCards.first(where: { $0.header.cardID == 12 }) else {
```

Comment:
Should this card be passed to the configure method and you check this logic before calling the function?

Reply:
Could be part of the refactoring

Reply:
@flxschmidt please comment to.do, so we don't forget?

Reply:
we don't need this. When we want to have a complete generic solution, based on card sequence array, you would go through the sequence and map the types of cards ...